src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,74 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type TabChangeCallback = (tab: Tab) => void;
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: TabChangeCallback;
+};
+
+const refineSelectedId = (tabs: Tab[], selectedTabId: string) => {
+  const isValidSelectedId = tabs.some(tab => tab.id === selectedTabId);
+
+  if (!isValidSelectedId) {
+    return tabs[0].id;
+  }
+
+  return selectedTabId;
+};
+
+const getSelectedTabContent = (tabs: Tab[], selectedId: string) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedId);
+
+  return selectedTab && selectedTab.content;
+};
```

Comment:
why you need these 2 func?
```suggestion
const getSelectedTabById = (tabs: Tab[], id: string) => (
  tabs.find(tab => tab.id === selectedId) || tabs[0];
);
```